File: features/snippets/external_storage/threshold/threshold_config.py
import dataclasses

from temporalio.converter import DataConverter, ExternalStorage


def configure(driver):
    # @@@SNIPSTART python-external-storage-threshold
    data_converter = dataclasses.replace(
        DataConverter.default,
        external_storage=ExternalStorage(
            drivers=[driver],
            payload_size_threshold=0,
        ),
    )
    # @@@SNIPEND
    return data_converter

File: features/snippets/external_storage/threshold/threshold_config.ts
import { ExternalStorage, type StorageDriver } from '@temporalio/common';
/* eslint-disable @typescript-eslint/no-unused-vars */

declare const driver: StorageDriver;

{
  // @@@SNIPSTART typescript-external-storage-threshold
  const dataConverter = {
    externalStorage: new ExternalStorage({
      drivers: [driver],
      payloadSizeThreshold: 0,
    }),
  };
  // @@@SNIPEND
}
